src/Carbon/CarbonMod.cs

```diff
@@ -85,12 +85,8 @@ private static void GetCarbonAssets(orig_SplitName orig, string name, out string
 		{
 			if (name.StartsWith("TeamCatalyst/Carbon/Module"))
 			{
-																			// TeamCatalyst/Carbon/Module/ModuleName/...
-				string subname = name.Substring(name.IndexOf("/") + 1);     // Carbon/Module/ModuleName/...
-				subname = subname.Substring(subname.IndexOf("/") + 1);      // Module/ModuleName/...
-				subname = subname.Substring(subname.IndexOf("/") + 1);      // ModuleName/...
-				subname = subname.Substring(subname.IndexOf("/") + 1);      // ...
-				name = $"Carbon/{subname}";
+				var split_path = name.Split("/");
```

Comment:
splitPath

Reply:
Also add a guard to avoid crashing when someone uses an impossible path (.e.g. `TeamCatalyst/Carbon/Module`, which would complete the check and throw and IOOB.